src/main/java/com/adm/utils/uft/integration/JobOperation.java

```diff
@@ -117,10 +118,9 @@ public String upload(String appPath) throws HttpConnectionException, IOException
                 jsessionId = (String) jsonObject.get(JSESSIONID);
             }
         } catch (Exception e) {
-            e.printStackTrace();
+            return null;
```

Comment:
don't you like to print the error , or it enough to print the error on calling method.
you may print here fail parsing

Reply:
In my opinion if the called method is an operation it is the operation caller's responsibility to report its result. A more elegant way would be to rethrow the exception and to handle that, that way we could attach a message to this error. However, this return null should not appear as the login should always return at least an empty JSONObject. The null check is here for a safety measure and because of the fact that the ParseException is not runtime.